code/game/objects/items/explosives/grenades/marines.dm

```diff
@@ -420,3 +420,26 @@
 /obj/item/explosive/grenade/flare/cas/turn_off()
 	QDEL_NULL(target)
 	return ..()
+
+///Flares that the tadpole flare launcher launches
+/obj/item/explosive/grenade/flare/strongerflare
+	icon_state = "stronger_flare_grenade"
+	lower_fuel_limit = 25
+	upper_fuel_limit = 30
+	light_system = STATIC_LIGHT//movable light has a max range
+	///The brightness of the flare
+	var/brightness = 12
+	light_color = LIGHT_COLOR_CYAN
```

Comment:
Use var ordering properly

Also isnt there a var for this already

Reply:
nope the normal flare uses light_range